pkg/multicluster/target.go

```diff
@@ -111,7 +111,7 @@ func (t *ClusterGatewayTarget) GetName() string {
 }
 
 func (t *ClusterGatewayTarget) GetShortCode() string {
-	return ToBase36hash(t.GetName())
+	return fmt.Sprintf("%s-%s", t.ClusterName, ToBase36HashLen(t.Namespace+"-"+t.Name, 7))
 }
```

Comment:
Does this not need to include the Gateway name, or are we saying per namespace is unique enough? This is being called [here ](https://github.com/Kuadrant/kuadrant-operator/pull/493/files#diff-75ea4f4ad086ceff450395450ca7845c9507114178820b7e625f9221c0a893b1R234) to generate the unique cluster specific endpoint, so it's just using `<clusterName>-<hash of namespace>`, where i thought we were doing `<clusterName>-<GW NS>-<GW Name>` 

Also, since you aren't hashing the result of `GetName`now, cluster names via a control plane (OCM) won't be hashed and will appear in plain text.